WORKSPACE

```diff
@@ -38,8 +49,8 @@ new_git_repository(
 new_git_repository(
     name = "plasma",
     build_file = "@//bazel:BUILD.plasma",
-    commit = "6a27c660ea700febf6fd73b2e851ab96e9315134",
-    remote = "https://github.com/ray-project/arrow",
+    commit = "9c33e1aee98f96a82c6a0b6bb214df172a143fb1",
+    remote = "https://github.com/apache/arrow",
```

Comment:
@pcmoritz we set plasma to the latest master of the apache repo here, which includes your dlmalloc fix. And we've set default plasma memory size to 100mb for tests. So CI won't trigger the leak issue. 

what's the status of #4385? If the leak issue is hard to fix, can we merge this PR first?

Reply:
Given that https://github.com/ray-project/ray/pull/4385 is almost ready to merge now, I think it makes sense to wait for it, since then both the pyarrow wheels and the bazel sources will be in sync and we can avoid having a master with the leak. If we can get #4385 merged tomorrow let's wait for it and otherwise I'm happy to merge this PR first.

Reply:
ok, let's wait till tomorrow then.